src/components/ui/common/videocard.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { VideocardProps } from "../../../lib/definitions";
+
+const VideoCard = ({ videoUrl, title, description, buttonText }:VideocardProps) => {
+  return (
+    <div className="bg-white dark:bg-foundationcolorwhite p-4 rounded-lg shadow-md">
+      <div className="aspect-w-16 aspect-h-9">
+        <iframe
```

Comment:
Can this Iframe be sandboxed?

```suggestion
        <iframe
                 sandbox="allow-scripts allow-same-origin allow-presentation"
```